### PR TITLE
Fix a typo in CONTRIBUTING.md

### LINKED ISSUE #903: Fix typo in CONTRIBUTING.md
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Todo: Correct a typo in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by therohitramesh**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-
-

Fixes #903

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-904-open-event-frontend-fossasia.surge.sh

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 **Commits**
 * Write clear meaningful git commit messages (Do read http://chris.beams.io/posts/git-commit/)
 * Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged. (More info at https://github.com/blog/1506-closing-issues-via-pull-requests )
-* When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
+* When you make very very minor changes to a PR of yours (like for example fixing a failing travis-build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
```
Comment (niranjan94) [inline .github/CONTRIBUTING.md]: I don't see a typo here.

Comment (therohitramesh) [reply .github/CONTRIBUTING.md]: I thought the correct way of writing it is travis-build. I'm sorry.

Comment (niranjan94) [reply .github/CONTRIBUTING.md]: No. travis build is right. no hyphen needed.

**Pull request closed without merging**